tasks/libs/common/githubapp.py

```diff
@@ -44,7 +44,7 @@ def get_headers(self):
         import jwt
 
         token_payload = self.gen_token_payload()
-        bearer_token = jwt.encode(token_payload, base64.b64decode(self.key_b64), algorithm='RS256').decode()
```

Comment:
I believe `jwt.decode` is used to decode a jwt. Here I think the bump only changes the return type of `jwt.encode` from byte array to string, which means that you should just drop the `.decode()` part